### PR TITLE
Output generated conversion functions/names

### PR DESCRIPTION
**Pull request by liggitt**
This outputs the generated functions/names to files so that people who hit test failures related to needing to regenerate conversion functions already have them generated

### CODE REVIEW

**Comment by liggitt** [comment]
@wojtek-t PTAL

**Comment by liggitt** [comment]
@pmorie FYI

**Comment by pmorie** [comment]
@pweil- @sdminonne

File: pkg/conversion/code_generation_test.go [hunk 1]
```diff
@@ -145,7 +146,13 @@ func TestNoManualChangesToGenerateConversions(t *testing.T) {
...
-		compareBuffers(t, existingFunctions, generatedFunctions)
-		compareBuffers(t, existingNames, generatedNames)
+		functionsTxt := fmt.Sprintf("%s.functions.txt", version)
+		ioutil.WriteFile(functionsTxt, generatedFunctions.Bytes(), os.FileMode(0644))
+
+		namesTxt := fmt.Sprintf("%s.names.txt", version)
+		ioutil.WriteFile(namesTxt, generatedNames.Bytes(), os.FileMode(0644))
+
+		compareBuffers(t, functionsTxt, existingFunctions, generatedFunctions)
+		compareBuffers(t, namesTxt, existingNames, generatedNames)
```
Comment (pweil-) [inline pkg/conversion/code_generation_test.go]: should probably clean up the files for a successful run (or maybe always).  If not, maybe add them to a gitignore

**Comment by pweil-** [comment]
One of the things I struggled with recently was that the generator output didn't tell me all of the things it failed to generate based on an api change I made (removing an field from internal but leaving it in the v1beta* apis).  It was on Container so pretty much everything that had a container didn't generate.  Maybe we can use a technique like this to give some output in the conversion utility for reference.

File: pkg/conversion/code_generation_test.go (hunk 1 above)
Comment (pweil-) [inline pkg/conversion/code_generation_test.go]: derp, the files are meant to be used.  Disregard...failed to comprehend.

Comment (pweil-) [inline pkg/conversion/code_generation_test.go]: But perhaps clean up in the success case still since they're only meant to be used in the failure case

**Comment by liggitt** [comment]
Cleaned up files if there were no problems

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**